wearApp/src/main/java/fr/paug/androidmakers/wear/data/LocalPreferencesRepository.kt

```diff
@@ -0,0 +1,11 @@
+package fr.paug.androidmakers.wear.data
+
+import android.content.Context
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.jraf.android.kprefs.Prefs
+
+class LocalPreferencesRepository(applicationContext: Context) {
+  private val localPrefs = Prefs(applicationContext)
+
+  val showOnlyBookmarkedSessions: MutableStateFlow<Boolean> by localPrefs.BooleanFlow(false)
```

Comment:
Is is possible to use the data store ? Like in the BookmarksDatastoreRepository : 
```
private val dataStore: DataStore<Preferences>

dataStore.data.map { prefs ->
        prefs[booleanPreferenceKey("MY_KEY")]
}
```

Reply:
Probably! 

Used my own little lib `kprefs` because I know it 😅. I'll have a look at `DataStore`, but I'll merge this one now to iterate fast.